modules/core/src/image-utils/image-loaders.js

```diff
@@ -26,17 +27,25 @@ function parseToImageBitmap(arrayBuffer) {
 }
 
 function loadToHTMLImage(url, options) {
-  return new Promise((resolve, reject) => {
+  let promise;
+  if (/\.svg((\?|#).*)?$/.test(url)) {
+    // is SVG
+    promise = readFile(url, {dataType: 'text'})
```

Comment:
Nit: We can use `async/await` now, makes the code prettier.

Reply:
Will address in a follow up PR.